docs/docs/reference/configuration.md

```diff
@@ -76,6 +76,27 @@ Defines the bootstrap server of Kafka brokers to consume from. Multiple Kafka br
 * Example value: `localhost:9092`
 * Type: `required` only when `KAFKA_CONSUMER` or `KAFKA_SOURCE` is configured in `SOURCE_DETAILS`
 
+##### `SOURCE_KAFKA_CONSUMER_CONFIG_SECURITY_PROTOCOL`
+
+Defines the security protocol used to communicate with ACL enabled kafka. Valid values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL.
+
```

Comment:
We should probably explain what each of these protocols mean. Or atleast link them to some trusted official documentation.